lib/vtls/schannel_verify.c

```diff
@@ -197,6 +197,7 @@ static CURLcode add_certs_data_to_store(HCERTSTORE trust_store,
                   actual_content_type, ca_file_text);
             result = CURLE_SSL_CACERT_BADFILE;
             more_certs = 0;
+            CertFreeCertificateContext(cert_context);
```

Comment:
What do you think about instead doing the CertFreeCertificateContext() call after both the if and the else, so that it can be done once for either condition? Roughly on line 222.